janitor/xarray/utils.py

```diff
@@ -0,0 +1,43 @@
+"""
+Helpers to facilitate creating new XArray methods
+"""
+
+
+import xarray as xr
+
+from xarray import register_dataarray_accessor, register_dataset_accessor
+from functools import wraps
+
+
+def make_accessor_wrapper(method):
```

Comment:
@zbarry just a small suggestion, I think it can be cleaner to push it to [pandas_flavor](https://github.com/Zsailer/pandas_flavor) I don't mind taking that task :).

Reply:
@eyaltrabelsi - sounds great. Thanks!

Reply:
@zbarry It was merged to pandas_flavor :)

Reply:
:+1: Thanks!

Reply:
Removed my method registration code and moved to using pf’s